changelog.json

```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "2.2.1-beta1": [
+      "[Fixed] Improperly formatted url pre-filled in \"Clone Repository\" dialog - #8394",
+      "[Fixed] VSCodium integration on macOS - #8424. Thanks @akim8!",
+      "[Fixed] Partially hidden icon in \"Open Pull Request\" button - #8410",
+      "[Improved] Ensure crash reports are sent when app closes unexpectedly - #8405"
```

Comment:
would we consider this an externally facing change?

Reply:
ooo that's a good point. i'll remove it

Reply:
```suggestion
```